sre_yield/tests/test_sre_yield.py

```diff
@@ -47,12 +49,21 @@ def testOtherCases(self):
         )
         self.assertEqual(len(sre_yield.AllStrings("..", charset="0123456789")), 100)
         self.assertEqual(len(sre_yield.AllStrings("0*")), 65536)
+        self.assertEqual(sre_yield.AllStrings("[01]*").__len__(), 2 ** 65536 - 1)
+
+    def testOverflowError(self):
         # For really big lists, we can't use the len() function any more
-        with self.assertRaises(OverflowError):
+        with self.assertRaises(OverflowError) as cm:
```

Comment:
assertRaisesRegex can do the message check in one line

Reply:
As the message strings are long, the single line syntax doesnt fit in a single line - and can look quite messy in a `with`.

Also the messages often change over time, while the Exception subclass and trigger logic usually does not.